schemas/preferences.sublime-settings.json

```diff
@@ -1211,6 +1211,47 @@
           "type": "array",
           "items": {"type": "string"},
           "default": ["Vintage"]
+        },
+        "default_extension": {
+          "markdownDescription": "**Unlisted.** Set's the default extension in the OS Save dialog when an untitled file is saved.",
+          "type": "string",
+          "default": ""
+        },
+        "show_legacy_color_schemes": {
+          "markdownDescription": "**Unlisted.** Show's the legacy color schemes in UI: Select Color Scheme. Added in Build 3136.",
+          "type": "boolean",
+          "default": false
+        },
+        "gtk_enable_key_translation": {
+          "markdownDescription": "**Unlisted.** Whether ST4's new key translation behavior should be enabled. Added in Build 4103.",
+          "type": "boolean",
+          "default": true
+        },
+        "gtk_client_side_window_decorations": {
+          "markdownDescription": "**Unlisted.** To disable the client side window decorations when using GNOME on Linux.",
+          "type": "boolean",
+          "default": false
+        },
+        "auto_complete_enable_plugins": {
+          "markdownDescription": "**Unlisted.** Disable auto completions from (all third party) plugins even when explicitly requested.",
+          "type": "boolean",
+          "default": false
+        },
+        "print_using_browser": {
+          "markdownDescription": "**Unlisted.** Print's the active view using the given browser as described by [webbrowser.get](https://docs.python.org/3/library/webbrowser.html#webbrowser.get) API. Added in Build 4094.",
+          "oneOf": [
+            {
+              "type": "null",
+            },
+            {
+              "type": "string"
+            }
+          ]
+        },
+        "default_dir": {
+          "markdownDescription": "**Unlisted.** The folder location where the Save As OS dialog should open when saving untitled new files.\nShould be an absolute path on disk",
```

Comment:
A single newline doesn't do anything in the used markdown parser. You need two newlines if if you want to break the line.
Also could add a trailing period.

Reply:
Done (for this and all the corrections mentioned below)